### PR TITLE
Add expanded row content to project list

### PR DESCRIPTION
**Pull request by marshmalien**
##### SUMMARY
#5070 

Add the following details to expanded area: 
* Description
* Organization
* Execution Environment
* Last modified
* Last used

Screen Shot 2021-04-16 at 2 03 35 PM

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

 - UI

### CODE REVIEW

**Review by nixocio — APPROVED** [review]

**Comment by nixocio** [comment]
I triggered an e2e tests.

**Comment by tiagodread** [comment]
> I triggered an e2e tests.

1 broken test potentially by this change, looking

**Review by tiagodread — COMMENTED** [review]
@marshmalien just one question, do we have unit test cov for this change? 
e2e test fixed in 6239

File: awx/ui_next/src/screens/Project/ProjectList/ProjectListItem.test.jsx [hunk 1]
```diff
@@ -319,4 +319,71 @@ describe('<ProjectsListItem />', () => {
     ).toBe('Sync for revision');
     expect(wrapper.find('ClipboardCopyButton').prop('isDisabled')).toBe(true);
   });
+  test('should render expected details in expanded section', async () => {
```
Comment (nixocio) [inline awx/ui_next/src/screens/Project/ProjectList/ProjectListItem.test.jsx:322]: 👍

**Comment by marshmalien** [comment]
@tiagodread I pushed a second commit with unit test coverage.

**Review by tiagodread — APPROVED** [review]
👍🏽

**Pull request merged**